src/openapi.json

```diff
@@ -744,6 +803,88 @@
 				}
 			}
 		},
+		"/organizationProposals": {
+			"get": {
+				"operationId": "getOrganizationProposals",
+				"summary": "Gets a list of OrganizationProposals.",
+				"tags": ["Organization Proposals"],
+				"security": [
+					{
+						"auth": []
+					}
+				],
+				"parameters": [
+					{ "$ref": "#/components/parameters/pageParam" },
+					{ "$ref": "#/components/parameters/countParam" },
+					{ "$ref": "#/components/parameters/organizationParam" },
+					{ "$ref": "#/components/parameters/proposalParam" }
+				],
+				"responses": {
+					"200": {
+						"description": "The requested OrganizationProposals.",
+						"content": {
+							"application/json": {
+								"schema": {
+									"$ref": "#/components/schemas/OrganizationProposal"
```

Comment:
Ooh, I actually caught something!

The Swagger spec claims `/organizationProposals` returns just the `OrganizationProposal` entity directly, but the real API call wraps the entities in an `entries`, with a `total`. I compared the example schema for `OrganizationProposal` with `Proposal` and `Organization`, and those latter two examples both showed the correct structure, e.g.:

```json
{
  "entries": [
    {
      "id": 3407,
      "employerIdentificationNumber": "11-1111111",
      "name": "Example Inc.",
      "createdAt": "2024-03-27T20:08:23.839Z"
    }
  ],
  "total": 20
}
```

Reply:
Ah, I see. Those have `Bundle`s. I guess we need an `OrganizationProposalBundle` then?

Reply:
OOOOOOOoooooooo finally, this PR produces something worthswhile

Reply:
Fixing this, but also realizing... It looks like the "allOf" hack for 3.0 solves the `readOnly` issue but flattens `organization` to be of type string.

I think we need to call that "will be fixed by #849" and allow it.

Reply:
Oh nooooo, now I see what it is. That's bad! 😣 But yes, I think overall it's better the outer structure of the example schema be correct (and to have the `readOnly` fix) and that we just… try to get #849 on the board.